### PR TITLE
add error message for uk virtual account

### PR DESCRIPTION
**Pull request by mahdiyeh-deriv**

### CODE REVIEW

File: src/javascript/app/base/binary_loader.js [hunk 1]
```diff
@@ -141,19 +141,18 @@ const BinaryLoader = (() => {
...
         }
         if (config.no_mf && Client.isLoggedIn() && Client.isAccountOfType('financial')) {
-            BinarySocket.wait('authorize').then((res) => {
-                if (Client.isOfferingBlocked(res.authorize.country)) {
-                    displayMessage(error_messages.offerings_blocked());
-                } else {
-                    displayMessage(error_messages.no_mf());
-                }
+            BinarySocket.wait('authorize').then(() => {
+                displayMessage(error_messages.no_mf());
```
Comment (mamali-fs) [inline src/javascript/app/base/binary_loader.js]: Possible to move the entire if statement inside the `wait` call back on line 149?

Comment (mahdiyeh-deriv) [reply src/javascript/app/base/binary_loader.js]: yes, sure

File: src/javascript/app/pages/trade/tradepage.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const TradePage = (() => {
         }
         BinarySocket.wait('authorize').then(() => {
             const country = State.getResponse('authorize.country') || State.getResponse('website_status.clients_country');
-
+
```
Comment (mamali-fs) [inline src/javascript/app/pages/trade/tradepage.js]:
```suggestion

```

**Review by mamali-fs — DISMISSED** [review]
LGTM

**Review by mamali-fs — APPROVED** [review]

**Pull request merged**